add desktop navigation menu

fickryiman opened the pull request:
- add desktop navigation menu
- add transition and transform on menu hover

fickryiman committed:
add desktop navigation menu

fickryiman committed:
fixes stylelint

fickryiman committed:
add button start collaboration animation and transition

fickryiman committed:
fixes stylelint (linter)

fickryiman committed:
add projects section

fickryiman committed:
fixes mobile version while add desktop version

fickryiman committed:
fixes stylelint

fickryiman committed:
fixes stylelint

fickryiman committed:
fixes stylelint

fickryiman committed:
fixes stylelint

fickryiman committed:
add project images

fickryiman committed:
finished project section

fickryiman committed:
fixes stylelint

fickryiman committed:
fixes stylelint

fickryiman committed:
add about section

fickryiman committed:
fixes stylelint

fickryiman committed:
finished the projects

wayungi commented on styles.css:
```diff
@@ -2,12 +2,589 @@ body,
+/* end of  about section
+
+/* mobile version */
+@media screen and (max-device-width: 767px),
```
- [x] Please note that we are building based on a mobile-first approach. Your media query should be based on min-width and not max-width. `@media screen and (min-width: value px) { }` 🪖

fickryiman replied:
noted, will take much time to fixes, otherwise i notice and learn from this, should be mobile-first approach not desktop-first approach.

wayungi reviewed (changes requested):
Hi @fickryiman , 
![image](https://user-images.githubusercontent.com/99656687/220082777-7a0e6a7f-854d-4e32-9c38-3b1eb3b090e4.png)
Very good work so far ! 🤝 
There are some issues that you still need to work on to go to the next project but you are almost there! 👯 

## Highlights 💡 💡 
- [x] All linter checks passing ✔️ 
- [x] Good commit messages ✔
- [x] No merge conflicts ✔️ 
- [x]  Correct GitHub flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [x] Please fix the styling of your contact form to match the design. See discrepancies below.
![from1](https://user-images.githubusercontent.com/99656687/221364717-b8b89d1f-baaf-42c8-8a1a-8aadb68c8e50.png)
**Your design**
![form2](https://user-images.githubusercontent.com/99656687/221365048-792a9093-a9fa-4e9f-8f41-817cb4adbbf8.png)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fickryiman committed:
fixes after review

fickryiman commented:
I have finished and fixed all the reviewers suggested.

fickryiman committed:
fixes stylelint

huseyinbiyik commented on styles.css:
```diff
@@ -2,12 +2,678 @@ body,
+
+/* mobile version */
+@media screen and (max-device-width: 767px),
+  screen and (max-width: 767px) {
```
As the previous reviewer mentioned, please use min-width value only since we follow mobile first approach. In this way, you will use less code for different screen sizes and your web pages will be more adaptive.

fickryiman replied:
changes to mobile first approach

huseyinbiyik commented on styles.css:
```diff
@@ -2,12 +2,678 @@ body,
+  padding-left: 20vw;
+}
+
+/* start about section desktop version */
```
Please don't leave any uncommented code in your codebase. In this way, you can keep your codebase clean, organized and professional 👍

fickryiman replied:
remove comment code, to keep codebase clean, fixes

huseyinbiyik commented on index.html:
```diff
@@ -110,54 +140,62 @@ <h3 class="project-name">Project name goes here</h3>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</h3>
       </div>
 
       <!-- form start -->
```
As the previous reviewer requested, please follow the Figma template and
- Please make sure label are aligned with the input areas. And text area has the total width of the name and email areas.
<img width="1440" alt="Screenshot 2023-02-25 at 8 23 06 PM" src="https://user-images.githubusercontent.com/25887253/221371148-4ce3183b-7bd1-4569-94be-a1408e6512e3.png">
<img width="593" alt="Screenshot 2023-02-25 at 8 34 14 PM" src="https://user-images.githubusercontent.com/25887253/221371239-f78b0a16-b213-47bd-966a-0a1e2d01a08d.png">

fickryiman replied:
fixes

OyePriscilla replied:
- [x] Kindly remove the **`padding-left`** attribute for the three labels, so that your design will be according to the given template. 

## Your Design 
<img width="685" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/221381344-77cb58f7-962d-4808-a578-47a903ad42be.png">

fickryiman committed:
fixes minor UI for more beautiful

huseyinbiyik commented on index.html:
```diff
@@ -193,28 +231,28 @@ <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod temp
             </textarea>
           </div>
         </div>
         <button type="submit" class="button-get-started">Start collaboration</button>
```
As the previous reviewer requested, please follow the Figma template and fix the width of the submit button.
<img width="947" alt="Screenshot 2023-02-25 at 8 35 24 PM" src="https://user-images.githubusercontent.com/25887253/221371488-f917d7da-b652-4a5f-92df-6cb94b7b6ffe.png">
<img width="794" alt="Screenshot 2023-02-25 at 8 37 34 PM" src="https://user-images.githubusercontent.com/25887253/221371489-675ea302-527e-4fa8-89ac-d2686fc8db18.png">

fickryiman replied:
fixes

huseyinbiyik reviewed (changes requested):
Hi @RHK-MICROVERSE,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![tenor](https://user-images.githubusercontent.com/25887253/205415161-00e79d1b-af68-4f10-9b2c-265f460dc5da.gif)

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fickryiman committed:
fixes as reviewer suggestions

- changes from desktop-first approach to mobile-first approach
- clean code (remove uncomment code)
- fixes positions

fickryiman commented:
need extra efforts for changing from desktop-first approach to mobile-first approach

fickryiman committed:
fixes stylelint

fickryiman committed:
fixes stylelint

fickryiman committed:
fixes stylelint

OyePriscilla commented on styles/mobile-version.css:
```diff
@@ -0,0 +1,751 @@
+/* mobile version */
+
+/* @media screen and (max-device-width: 767px),
+  screen and (max-width: 767px) { */
```
- [x] This is a typical example of another commented code in your project, please kindly remove it as suggested earlier that you should not leave **any** commented code in your codebase. Observing this will make your codebase clean. ℹ️

fickryiman replied:
noted, this is because i have to finish it fast due to last minute assignment. fixes done

OyePriscilla commented on styles/mobile-version.css:
```diff
@@ -0,0 +1,751 @@
+}
+
+.projects-section > div:nth-child(4) > .project-info {
+  /* display: flex;
+  flex-direction: column; */
```
- [x] This is another commented code, please kindly remove it as explained earlier. 👍

fickryiman replied:
fixes done

OyePriscilla commented on index.html:
```diff
@@ -7,13 +7,21 @@
-  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap" rel="stylesheet">
+  <!-- <link rel="stylesheet" href="styles.css"> -->
```
- [x] This is a commented code as well. Kindly remove it as suggested earlier. 👍

fickryiman replied:
fixes done

OyePriscilla commented on styles/desktop-version.css:
```diff
@@ -0,0 +1,632 @@
+  }
+
+  .button-get-started {
+    width: 50%;
```
- [x] I think you may not necessarily use a relative **width** for the contact form `button` element, kindly use `254px` (for the desktop version) as suggested earlier according to the given template ℹ️

fickryiman replied:
fixes done

OyePriscilla reviewed (changes requested):
Hi @fickryiman 👏 , 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425724-8ca5825c-b02a-4983-b976-4bd0a2f70f9a.gif)

 ## Highlights
Use of CSS media query ✔️ 
CSS Flexbox and Grid to layout elements :mark#
Good communication with code reviewer ✔️ 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fickryiman commented:
i have finished all required changes, otherwise i need to extra review request

fickryiman committed:
remove all commented code and remove padding left on form

fickryiman committed:
change bottom padding top, padding bottom
